src/SDKs/ServiceFabric/Management.ServiceFabric/Generated/ClusterVersionsOperations.cs

```diff
@@ -78,7 +289,7 @@ internal ClusterVersionsOperations(ServiceFabricManagementClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<IPage<ClusterCodeVersionsResult>>> ListWithHttpMessagesAsync(string location, string environment, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<ClusterCodeVersionsListResult>> GetByEnvironmentWithHttpMessagesAsync(string location, string environment, string clusterVersion, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
@juhacket Still looks like renaming a method, looks like this can affect current users

Reply:
Yes, we changed the model for our paged objects. I have updated the version to reflect the intent to make this breaking change.